rust/src/operations/transaction/mod.rs

```diff
@@ -68,9 +71,25 @@ impl From<TransactionError> for DeltaTableError {
 // Convert actions to their json representation
 fn log_entry_from_actions<'a>(
     actions: impl IntoIterator<Item = &'a Action>,
+    read_snapshot: &DeltaTableState,
 ) -> Result<String, TransactionError> {
+    let append_only = read_snapshot.current_metadata().map_or(false, |meta| {
+        meta.configuration
+            .get("delta.appendOnly")
```

Comment:
```suggestion
            .get(DeltaConfigKey::AppendOnly)
```

Since we already have this constant defined elsewhere, it would make sense to me to just refer to it

Reply:
I would go a bit further - hidden ion the depth of our code, we have a `TableConfig` type, that abstracts reading table configuration. Its accessible via `table_config()` on the snapshot.

Reply:
Updated.